enhancements/node-tuning/mixed-cpu-node-plugin.md

```diff
@@ -0,0 +1,322 @@
+---
+title: mixed-cpu-node-plugin
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@MarSik"
+  - "@yanirq"
+  - "@ffromani"
+  - "@jmencak"
+  - "@mrunalp"
+approvers: 
+  - "@mrunalp"
+api-approvers:
+  - "@mrunalp"
+creation-date: 2023-05-03
+last-updated: 2023-05-08
+tracking-link: 
+  - https://issues.redhat.com/browse/CNF-7603
+---
```

Comment:
Should probably add a see-also link to reference enhancements/shared-and-guaranteed-cpu-pools.md